test/unit/tracks/text-track-controls.test.js

```diff
@@ -16,236 +18,266 @@ const track = {
+      QUnit.ok(player.textTracks()[i].kind === 'descriptions' &&
+               player.textTracks()[i].mode === 'showing',
+               'descriptions mode set to showing');
     }
   }
 
   this.clock.tick(1000);
 
-  ok(!player.controlBar.captionsButton.hasClass('vjs-disabled'), 'captions control is NOT disabled');
-  ok(!player.controlBar.descriptionsButton.hasClass('vjs-disabled'), 'descriptions control is NOT disabled');
+  QUnit.ok(!player.controlBar.captionsButton.hasClass('vjs-disabled'),
+           'captions control is NOT disabled');
+  QUnit.ok(!player.controlBar.descriptionsButton.hasClass('vjs-disabled'),
+           'descriptions control is NOT disabled');
 
   for (let i = 0; i < player.textTracks().length; i++) {
     if (player.textTracks()[i].kind === 'captions') {
       player.textTracks()[i].mode = 'showing';
-      ok(player.textTracks()[i].kind === 'captions' && player.textTracks()[i].mode === 'showing', 'captions mode set to showing');
+      QUnit.ok(player.textTracks()[i].kind === 'captions' &&
+               player.textTracks()[i].mode === 'showing',
+               'captions mode set to showing');
     }
   }
 
   this.clock.tick(1000);
 
-  ok(!player.controlBar.captionsButton.hasClass('vjs-disabled'), 'captions control is NOT disabled');
-  ok(player.controlBar.descriptionsButton.hasClass('vjs-disabled'), 'descriptions control IS disabled');
+  QUnit.ok(!player.controlBar.captionsButton.hasClass('vjs-disabled'),
+           'captions control is NOT disabled');
+  QUnit.ok(player.controlBar.descriptionsButton.hasClass('vjs-disabled'),
+           'descriptions control IS disabled');
 
   for (let i = 0; i < player.textTracks().length; i++) {
     if (player.textTracks()[i].kind === 'captions') {
       player.textTracks()[i].mode = 'disabled';
-      ok(player.textTracks()[i].kind === 'captions' && player.textTracks()[i].mode === 'disabled', 'captions mode set to disabled');
+      QUnit.ok(player.textTracks()[i].kind === 'captions' &&
+               player.textTracks()[i].mode === 'disabled',
+               'captions mode set to disabled');
     }
   }
 
   this.clock.tick(1000);
 
-  ok(!player.controlBar.captionsButton.hasClass('vjs-disabled'), 'captions control is NOT disabled');
-  ok(!player.controlBar.descriptionsButton.hasClass('vjs-disabled'), 'descriptions control is NOT disabled');
+  QUnit.ok(!player.controlBar.captionsButton.hasClass('vjs-disabled'),
+           'captions control is NOT disabled');
+  QUnit.ok(!player.controlBar.descriptionsButton.hasClass('vjs-disabled'),
+           'descriptions control is NOT disabled');
 
   player.dispose();
 });
 
 if (!browser.IS_IE8) {
   // This test doesn't work on IE8.
-  // However, this test tests a specific with iOS7 where the TextTrackList doesn't report track mode changes.
+  // However, this test tests a specific with iOS7 where
```

Comment:
only urls are exempt, so I think that means only urls at the end of a line that run way over